docs/DevGuide/Observers.md

```diff
@@ -66,8 +66,8 @@ the observers. The action `observers::Actions::ContributeReductionData` is used
 to send data to the `observers::Observer` component in the case where there is a
 reduction done across an array or subset of an array. If a singleton parallel
 component needs to write data directly to disk it should use the
-`observers::ThreadedActions::WriteReductionData` action called on the zeroth
-element of the `observers::ObserverWriter` component.
+`observers::ThreadedActions::WriteSingletonReductionData` action called on the
```

Comment:
Now that singletons don't need to register in order to write data, I believe the `observers::Actions::RegisterSingletonWithObserverWriter` action from a few lines above will cause confusion. Maybe remove it from the list or make a note here that you don't *have* to register the singleton.